scanomatic/ui_server_data/analysis.html

```diff
@@ -2,17 +2,12 @@
 <html>
 <head lang="en">
   <meta charset="UTF-8">
-  <link rel="stylesheet" type="text/css" href="style/main.css?ver=_-_VERSIONTAG_-_">
-  <link rel="stylesheet" type="text/css" href="style/analysis.css?ver=_-_VERSIONTAG_-_">
-  <script src="/js/external/jquery.js"></script>
-  <script src="/js/external/jquery-ui.js"></script>
-  <script src="/js/helpers.js?ver=_-_VERSIONTAG_-_"></script>
-  <script src="/js/fixtures.js?ver=_-_VERSIONTAG_-_"></script>
-  <script src="/js/analysis.js?ver=_-_VERSIONTAG_-_"></script>
-  <script src="/js/calibration.js?ver=_-_VERSIONTAG_-_"></script>
+  <script src="/js/somlib/som.js?ver=_-_VERSIONTAG_-_"></script>
```

Comment:
here you have the version tag thing on `som.js`, but in the two previous files it's not. Don't know if it's needed but I think it at least should be systematic.

Reply:
I have retained it where it was originally, but some files lacked it. I didn't want to change this, in case there was a thought behind this, but I can add or remove it everywhere if you want. Note that one file (`qc_norm.html`) sets version explicitly to 1.0.0. Anyway, let me know what you want me to do about this.

Reply:
if the ccc file import which is made in the same way had it, then add it to all, if the ccc file doesn't then remove it from all som.js

Reply:
`ccc.js` does not have version, so I will remove it from all.

Reply:
I noted that the css-links are also inconsistently handled, some have the version tags, others do not. I suggest I remove them while I'm at it, since I have to go through all the files anyway.

Reply:
fixed (css are now included in #128)